### ISSUE TITLE
[Feature]: Adding global html attributes to DocumentCard components

### ISSUE THREAD

**Issue by njeri-ngigi** [microsoft/fluentui#30324]
### Library

React / v8 (@fluentui/react)

### Describe the feature that you would like added

Could you add global html attributes such as id to the DocumentCard components: DocumentCard, DocumentCardDetails, DocumentCardPreview, DocumentCardTitle, DocumentCardType.

Maybe extending the HTMLAttributes.

The reason for the ask is to be able to add accessibility aria elements such as aria-labelledby, aria-labelledby which rely on ids added to components.

### Additional context

<img width="1057" alt="Screenshot 2024-01-16 at 15 56 43" src="https://github.com/microsoft/fluentui/assets/28973383/2246de3b-ad8a-49f2-a833-a913de516c1d">

Also started a discussion on stackoverflow about the same. Follow [here](https://stackoverflow.com/questions/77826096/fluent-ui-documentcardtitle-global-html-attributes)

### Validations

- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### Priority

Normal

**Comment by tudorpopams** [comment]
Hello @njeri-ngigi , thank you for reaching out. We are not planning on doing new feature requests on v8 any more as we recommend the [v9 Card alternative](https://react.fluentui.dev/?path=/docs/components-card-card--default). Let us know if this might be a solution for you.

@smhigley is this v8 issue something that can already be achieved and I'm not aware of?

**Comment by smhigley** [comment]
@njeri-ngigi thanks for the issue! The root `DocumentCard` component accepts miscellaneous HTML `<div>` props, so you should be able to add an `id` and `aria-*` props there.

For the others, unfortunately as @tudorpopams said, we're not doing new feature work on v8. I'd recommend using `aria-label` strings instead of id references for now, and transitioning to the v9 Card when you're able to.
